filter.py

```diff
@@ -0,0 +1,67 @@
+from pymongo import MongoClient
+import pandas as pd
+
+
+# Dictionary of NTEE codes
+# TODO: Add all NTEE codes with service summaries
+NTEE_CD = {
+  "K31": "Food Pantry",
+  "154": "Health Clinic"
+}
+
+
+def read_csv(ntee_cd):
+  """
+  Filter a CSV by NTEE code.
+
+  In:
+  ntee_cd (str): NTEE code.
+
+  Out:
+  (DataFrame): Filtered CSV as Pandas data frame.
+  """
+  data = pd.read_csv("./Mid-Atlantic and Great Lakes Areas.csv", usecols = ["NAME", "STREET", "CITY", "STATE", "ZIP", "NTEE_CD"])
+  filter_data = data["NTEE_CD"] == ntee_cd
+  req_data = data[filter_data]
+  return req_data
+
+
+def query_db(data, ntee_cd):
+  """
+  Add data to tmp storage if name and zip already exists in services,
+  otherwise add to services.
+
+  In:
+  data (DataFrame): Pandas data frame.
+  ntee_cd (str): NTEE code.
+  """
+  client = MongoClient("mongodb+srv://democracylab:DemocracyLab2019@shelter-rm3lc.azure.mongodb.net/test?authSource=admin&replicaSet=Shelter-shard-0&readPreference=primary&appname=MongoDB%20Compass&ssl=true")
```

Comment:
Just wanted to point out that - we wanted to avoid putting username and pwd in the public code. Looks great otherwise. Thank you.